File: Python Codes/CheckTriangle.py
a = int(input("Enter an side: "))
b = int(input("Enter an side: "))
c = int(input("Enter an side: "))
if a==b and b==c:
    print("Equilateral")
elif a==b or b==c:
    print("Isosceles")
else:
    print("Scalene")
#a+b>c
# b+c>a
# c+a>b

File: C program/New Batch/CheckTriangle.cpp
#include<stdio.h>
int main(){
    int a,b,c;
    printf("Enter 3 sides of a triangle: ");
    scanf("%d%d%d",&a,&b,&c);
    if(a==b && b==c){
        printf("Equilateral");
    }
    else if(a==b || b==c || a==c){
        printf("Isosceles");
    }
    else{
        printf("Scalene");
    }
    return 0;
}